product_docs/docs/pgd/5/reference/commit-scopes.mdx

```diff
@@ -147,7 +149,9 @@ When set to true, two-phase commit transactions:
 
 The value `async` means resolve conflicts asynchronously during replication using the conflict resolution policy.
 
-The value `eager` means that conflicts are resolved eagerly during COMMIT by aborting one of the conflicting transactions.
+The value `eager` means that conflicts are resolved eagerly during COMMIT by aborting one of the conflicting transactions. 
+
+When this option is used with the `ALL` commit scope group, the [`commit_decision`](#commit_decision-settings) must be set to `raft` to avoid reconcilation issue.
```

Comment:
Should we mention that `eager` can only be used with `MAJORITY` or `ALL` as well, i.e. the `ANY N` makes no sense with `eager`.

Reply:
Added notes to this.